public/data/shivamshrivastava.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Shivam Shrivastava",
+    "bio": "Python | ML | Flutter",
+    "avatar": "https://github.com/Shivam-Shrivastav.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "http://github.com/Shivam-Shrivastav",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "http://twitter.com/ShivamsTwt",
+        "icon": "twitter"
+      },
+      {
+        "name": "Learn more about Open Source on my YouTube channel",
```

Comment:
@Shivam-Shrivastav is this meant to display LinkedIn, not YouTube? Thanks.

Reply:
Okay I am gonna change it to LinkedIn, Thankyou.